go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -118,8 +118,12 @@ func LaunchCluster(setupType int, streamMode string, stripes int, cDetails *Comp
 	dbCredentialFile = cluster.WriteDbCredentialToTmp(localCluster.TmpDirectory)
 	initDb, _ := os.ReadFile(path.Join(os.Getenv("VTROOT"), "/config/init_db.sql"))
 	sql := string(initDb)
+	// Since password update is DML we need to insert it before we disable
+	// super-read-only therefore doing the split below.
+	splitString := strings.Split(sql, "# add custom sql here")
+	firstPart := splitString[0] + cluster.GetPasswordUpdateSQL(localCluster)
+	sql = firstPart + splitString[1]
```

Comment:
Same comment here about error checking and handling. This could result in a process crash.